seq2science/rules/configuration_generic.smk

```diff
@@ -201,6 +206,14 @@ def get_workflow():
     return workflow.snakefile.split('/')[-2]
 
 
+wf = get_workflow()\
+    .replace('alignment',  'Alignment')\
+    .replace('atac_seq',   'ATAC-seq')\
+    .replace('chip_seq',   'ChIP-seq')\
+    .replace('rna_seq',    'RNA-seq')\
+    .replace('scatac_seq', 'scATAC-seq')
```

Comment:
Maybe we should rename `wf` to `workflow` or sth. Not sure how DeScRiPtIvE it is :P

Anyways, regardless whether or not you change the name, please add it to:
https://github.com/vanheeringen-lab/seq2science/blob/master/docs/CONTRIBUTING.md#important-global-variables

Reply:
all valid points. 
 - wf renames to `sequencing_protocol` as workflow is taken by snakemake
 - I'll add it to the docs, but these aren't present in this PR, so later :)
 - I've quickly tried making this into a script, but there are a myriad of variables that this depends on (all_assemblies, has_annotation(), ori_assembly(), breps, treps, config...). It sounds like a nice challenge to set up. For another time ;)